pkgs/development/python-modules/bce-python-sdk/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, future
+, pycryptodome
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "bce-python-sdk";
+  version = "0.8.87";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "vgTf1HiD/R/1m24zq/vB+Dt2KzacfwePrfe1q0jB0qk=";
+  };
+
+  propagatedBuildInputs = [
+    future
+    pycryptodome
+    six
+  ];
```

Comment:
```suggestion
  ];
  
  # there is no tests
  doCheck = false;
```

Reply:
Ah, I thought that would also suppress the `pythonImportsCheck = [ "baidubce" ]`